release/nightly_tests/dataset/read_and_consume_benchmark.py

```diff
@@ -78,7 +78,7 @@ def consume_fn(ds):
     elif args.iter_torch_batches:
 
         def consume_fn(ds):
-            for _ in ds.iter_torch_batches():
+            for _ in ds.iter_torch_batches(device="cuda"):
```

Comment:
Move the testing of `iter_torch_batches` with CUDA from `streaming_data_ingest_benchmark_100gb_gpu` benchmark to the `iter_torch_batches` benchmark